Tools/.gitignore

```diff
@@ -4,3 +4,4 @@ csharpasyncgenerator.commandline/
 gitreleasemanager/
 obj/
 microsoft.*
+netstandard.library/
```

Comment:
In master we have instead:

```suggestion
netstandard.*
```

Wouldn't it better to align on it, or do I miss something?

Reply:
Yeah Ok. I didn't check master because it wasn't part of this change. And AsyncGenerator is used as global tool there it's most likely no longer needed